rskj-core/src/main/java/co/rsk/net/BlockSyncService.java

```diff
@@ -150,7 +150,23 @@ private void tryReleaseStore(long bestBlockNumber) {
         }
     }
 
+    public boolean isReadyToAcceptBlocks() {
+        if (blockchain.getSize() == 1) { // empty blockchain
+            return false;
+        }
+
+        if (getLastKnownBlockNumber() == 0) { // recently restarted, no info about lastKnownBlock yet
```

Comment:
This should return false here? We don't want new block hashes when we recently started, we are busy syncing.